consensus_test.go

```diff
@@ -465,8 +465,8 @@ func (m *mockPbft) emitMsg(msg *MessageReq) {
 
 func (m *mockPbft) addMessage(msg *MessageReq) {
 	// convert the address from the address pool
-	//from := m.pool.get(string(msg.From)).Address()
-	//msg.From = from
+	// from := m.pool.get(string(msg.From)).Address()
```

Comment:
Why is this being commented?

Reply:
This was already commented on in the main branch, and I did only formatting comments. Should I revert formatting?